woocommerce/wc-admin
# Scroll to top of the table when navigating table pages

Fixes #2014.

### Accessibility


### Screenshots
![table-scroll](https://user-images.githubusercontent.com/3616980/56044051-b0f96280-5d3e-11e9-8ade-1170682e8c91.gif)

### Detailed test instructions:
- Go to any report where there is table pagination.
- Try going to the next/previous table page.
- Verify you are scrolled to the top of the table.
- Try with different window sizes.
